server/models/buoy.py

```diff
@@ -22,3 +23,6 @@ class Buoy(Base):
     water_quality = Column(String(1), default="n", nullable=False)
     dart = Column(String(1), default="n", nullable=False)
     seq = Column(SmallInteger, nullable=True)  # tao_seq
+
+    meteorological = relationship("meteorological")
+    wave = relationship("wave")
```

Comment:
Have you tested these relationships by running queries through the ORM?

I think you need to capitalize "meteorological" and "wave" to avoid syntax errors since these refer to the model names.

Reply:
Still need to test everything out. Have to build out endpoints and routes in order to test the DB instances live. But I think it points to the table names and not the classes. Though I could be wrong. Will let you know when everything breaks lol

Reply:
Haha... you got this! 😃